scripts/apitypes-funcs/templates.go

```diff
@@ -42,7 +42,7 @@ func (obj *{{ .Type }}) SetKonnectID(id string) {
 	obj.Status.Konnect.ID = id
 }
 
-{{ end }}
+{{- end }}
```

Comment:
```suggestion
{{ end -}}
```
This way we should get a newline between the `GetTypeName` comment and the `SetKonnectID` function.